src/test/e2e/tests/specs.test.ts

```diff
@@ -201,7 +201,7 @@ describe('Verified user', () => {
       await mockSessionPage.enterText('1', 'UserIdInput');
       await mockSessionPage.selectOption('UserType');
 
-      //If USE_PROTOTYPE is set then it goes to Heroku, therefore re-open to Subscription Management
+      //If USE_PROTOTYPE is set then it goes to Heroku, therefore re-open to UserSubscriptions Management
```

Comment:
Not sure why you've change every occurrence of the `Subscription` to `UserSubscriptions `? Can you please check your work and roll back where appropriate?